src/Operations/Memory.py

```diff
@@ -63,7 +65,7 @@ def access_width(cls):
     def _do_memory_access(self, address, state):
         value = state.stack.pop()
         return [
-            f"let (byte, _) = extract_lowest_byte({value})",
+            f"let (local byte, _) = extract_lowest_byte({value})",
             f"mstore8(offset={address}, byte=byte)",
```

Comment:
Does `byte` need to be `local`? It's used only once on the second line and can't be revoked

Reply:
It prevents range_check_ptr from being revoked.